packages/EasyCore/src/Doctrine/Dispatchers/DeferredEntityEventDispatcher.php

```diff
@@ -39,90 +44,123 @@ public function __construct(EventDispatcherInterface $eventDispatcher)
     public function clear(?int $transactionNestingLevel = null): void
     {
         if ($transactionNestingLevel !== null) {
```

Comment:
Why don't you clear the `$this->entityInsertions` and `$this->entityUpdates` arrays for the specified transaction level?

Reply:
1. Now these fields don't contain transaction levels. These fields are needed only to choose the event type and construct it.
2. It's able to add transaction levels to these fields. But this will complicate the code, and I see no benefit for this (please correct me if I'm wrong).

Reply:
OK, but this way we need to add:
```
$this->entityInsertions = [];
$this->entityUpdates = [];
```
right before the early return in the `if` block.

Reply:
We can' add these lines before the early return in the `if` because [testEventsAreDispatchedWhenExceptionIsThrownAndCatched](https://github.com/eonx-com/easy-monorepo/blob/490ba6543df82f422bce9985659864290a68d625/packages/EasyCore/tests/Doctrine/Subscribers/EntityEventSubscriberTest.php#L68) will not pass.
Wrote a logic to clean up `entityInsertions`/`entityUpdates`.